packages/patternfly-4/react-core/src/components/Progress/ProgressContainer.js

```diff
@@ -42,20 +56,25 @@ const variantToIcon = {
   [ProgressVariant.success]: CheckCircleIcon
 };
 
-const ProgressContainer = ({ value, title, parentId, variant, measureLocation }) => {
+const ProgressContainer = ({ value, title, parentId, label, variant, measureLocation }) => {
   const StatusIcon = variantToIcon.hasOwnProperty(variant) && variantToIcon[variant];
   return (
     <Fragment>
-      <div className={css(styles.progressDescription)} id={`${parentId}-description`}>
+      <div className={css(progressDescription)} id={`${parentId}-description`}>
         {title}
       </div>
-      <div className={css(styles.progressStatus)}>
+      <div className={css(progressStatus)}>
+        <span className={css(progressMeasure)}>
+          {label}
+        </span>
         {(measureLocation === ProgressMeasureLocation.top || measureLocation === ProgressMeasureLocation.outside) && (
-          <span className={css(styles.progressMeasure)}>{value}%</span>
+          <span className={css(progressMeasure, measureLocation === ProgressMeasureLocation.top && progressLabel)}>
+            {value}%
+          </span>
```

Comment:
I feel that the label should override the value / measureLocation. Having to also set ProgressMeasureLocation.none doesn't feel clean, but don't feel strongly about it.